recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "4.8.6" %}
+{% set version = "4.10dev" %}
```

Comment:
How about 4.9.0? We don't like packaging dev versions unless there's a convincing argument to do so.